### ISSUE TITLE
Hi,Soumya Koduri, Which directory that the main test process relate?

### ISSUE THREAD

**Issue by AlexKingGo** [soumyakoduri/nfs-test-suite#3]
Hi Soumya Koduri,
I find that the pynfs module is an independent nfs test tool, 
and the files in test-dir directory together with files under nfs-test-suite are an independent part, I mean that these are two irrelevant  test tools because there arn't code dependence on pynfs in the files under test-dir. Do I understant right?

**Comment by soumyakoduri** [comment]
thats right.. almost all the tests used in this suite can be run independently. This project intention was to just bring them and to automate running all of them together.

In the other thread you have mentioned that your project is to simulate many NFS clients. Then "pynfs" is the test tool you may need to start with. It is a synthetic client tool which simulates NFS client and sends rpc requests to NFS server. The latest version of pynfs and how to use can be found below - 

#git clone git://linux-nfs.org/~bfields/pynfs.git
#cd pynfs
#yes  | python setup.py build
#cd nfs4.1
#./testserver.py  -v --outfile ~/pynfs.run.1 --maketree <server_ip>:/<export_share> --showomit --rundeps  all

(run ./testserver.py --help for more details about this command)

**Comment by AlexKingGo** [comment]
I see, thank you very much, आपका बहुत बहुत धन्यवाद.
आप आशीर्वाद दीजिए

**Comment by soumyakoduri** [comment]
you are welcome :)

**Comment by AlexKingGo** [comment]
Hi，Soumya Koduri.
    Sorry to bother you again.  Now I have a question, I want to simulate that the only one client mount many path by many TCP connection with one server. But I find os.system('mount XXX) will combine the TCP connections into one connection. On the otherhand, I use tcpdump and find two different instance of testserver.py  generate two different TCP connection. In the pynfs, I can't find test about client mount path, is there some function or examples in the pynfs/nfs4.1 related to mount?

regards.
Alex Zhang
